src/alert_rules/prometheus/prometheus_alerts.yaml

```diff
@@ -2,6 +2,19 @@
 - "name": "opensearch.alerts"
   "rules":
 
+  # Write requests rates
+  # =====================
+  - record: write:rejected_requests:rate2m
+    expr: rate(opensearch_threadpool_threads_count{name="write", type="rejected"}[2m])
+
+  - record: write:completed_requests:rate2m
+    expr: rate(opensearch_threadpool_threads_count{name="write", type="completed"}[2m])
+
+  # If there are no write rejections then we get can 0/0 which is NaN. This does not affect the
+  # OpenSearchWriteRequestsRejectionJumps alert
+  - record: write:reject_ratio:rate2m
+    expr: sum by (cluster, instance, node) (write:rejected_requests:rate2m) / on (cluster, instance, node) (write:completed_requests:rate2m)
```

Comment:
I believe the rate formula is not accurate here. The denominator should be "completed requests + rejected_requests" to have the right rate of failed requests